bee-ternary/src/t1b1.rs

```diff
@@ -5,6 +5,8 @@ use crate::{trit::ShiftTernary, Btrit, RawEncoding, RawEncodingBuf, Trit};
 
 use std::{hash, marker::PhantomData, ops::Range};
 
+const TRITS_PER_BYTE: usize = 1;
```

Comment:
Since it's now an associated constant, I think I wouldn't even go through another constant, this is a bit redundant